src/resources/Alteration/CooperativeResource.ts

```diff
@@ -8,11 +8,11 @@ export const CooperativeResource: ResourceIF = {
   displayName: GetCorpFullDescription(CorpTypeCd.COOP),
   nameRequestType: NameRequestEntityTypes.CP,
   addressLabel: 'Registered Office',
-  filingData: {
-    entityType: CorpTypeCd.COOP,
-    filingTypeCode: FilingCodes.ALTERATION,
+  filingData: [{
+    filingTypeCode: FilingCodes.CONVERSION,
```

Comment:
Wrong filing code. Should be ALTERATION.
Wrong entity type. Should be COOP.

Actually, just check all the ones you changed. Each resource file is specific to a filing type and entity type (which I can see by the path).

Reply:
I fixed all the ones I changed and made them correct sorry about that, that was a bone headed move.